lib/YaST/Kdump/FirmwareAssistedDumpPage.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Kdump Firmware Assisted Dump Main Page.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::Kdump::FirmwareAssistedDumpPage;
+use strict;
+use warnings;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init();
+}
+
+sub init {
+    my $self = shift;
+    $self->{cbox_fadump} = $self->{app}->checkbox({id => "\"use_fadump\"" });
+    $self->{btn_accept} = $self->{app}->button({id => 'next'});
```

Comment:
We have that NavigationBase.pm lib that covers the "next" button, so you could use 
`use parent 'Installation::Navigation::NavigationBase';`
and avoid putting extra code for the "next" button in this page.